pkg/harvester/edit/network.harvesterhci.io.vlanconfig/index.vue

```diff
@@ -314,12 +313,21 @@ export default {
       } else if (selector[HOSTNAME] && Object.keys(selector).length === 1) {
         const matchNode = allNodes.find(n => n.id === selector[HOSTNAME]);
 
-        this.matchingNodes = {
-          matched: 1,
-          total:   allNodes.length,
-          none:    false,
-          sample:  matchNode ? matchNode.nameDisplay : selector[HOSTNAME],
-        };
+        if (matchNode) {
```

Comment:
matchNode may become undefined. We should set this.matchingNodes back to zero object.